api_blog/views.py

```diff
@@ -0,0 +1,12 @@
+import sqlalchemy as sa
+from aiohttp import web
+
+from .models import Session
+
+
+async def index(request):
+    async with Session() as session:
+        async with session.begin():
+            qs = sa.select(1)
+            await session.execute(qs)
```

Comment:
Let's refactor it a little bit)
```suggestion
            await session.execute(sa.select(1))
```